packages/fluentui/docs/src/examples/components/Datepicker/index.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+import Usage from './Usage';
+
+const DatepickerExamples = () => (
+  <>
+    <Usage />
```

Comment:
I believe this should be under `<Types />

Reply:
Moved to Types :)